README.rst

```diff
@@ -16,10 +39,92 @@ Django lookup extensions use **NOT** query.
 * Documentation: https://django-lookup-extensions.readthedocs.io.
 
 
-Features
---------
+Installation
+------------
+
+To install Django lookup extensions, run this command in your terminal:
+
+.. code-block:: console
+
+    $ pip install django-lookup-extensions
+
+This is the preferred method to install Django lookup extensions, as it will always install the most recent stable release.
+
+If you don't have `pip`_ installed, this `Python installation guide`_ can guide
+you through the process.
+
+.. _pip: https://pip.pypa.io
+.. _Python installation guide: http://docs.python-guide.org/en/latest/starting/installation/
+
+Usage
+-----
+
+Setup
+^^^^^
+
+Add **'lookup_extensions'** to your **INSTALLED_APPS** setting.:
+
+.. code-block:: python
+
+    INSTALLED_APPS = [
+        ...
+        'lookup_extensions',
+        ...
+    ]
+
+AppConfig.ready imports lookups and register them using **django.db.models.fields.Field.register_lookup**.
+
+Making queries
+^^^^^^^^^^^^^^
+
+Use lookup like Django standard lookups.
+
+.. code-block:: python
+
+    ModelA.objects.filter(name__neexact='test name')
+
+Supported lookup types
+----------------------
+
+neexact is negate exact, others are similar.
```

Comment:
It seems that too less descriptions for keywords.
Does `ne-` prefix means negate of consequence words like 'exact', 'contains', and 
 does `nei-` prefix means same as `ne-` prefix with ignore cases?
Could every developers use this extensions without descriptions for `nei-` prefixes?